vspace/pom.xml

```diff
@@ -14,7 +14,9 @@
         <spring-data.version>Kay-SR8</spring-data.version>
         <spring-security-version>4.1.3.RELEASE</spring-security-version>
         <javers.version>3.10.1</javers.version>
-        <admin.password>admin</admin.password>
+        <admin.password>admin</admin.password>         
+        <username>staff</username>
+      	<password>staff</password> 
```

Comment:
you still need to use this in the users.properties file, or this won't do anything